README.md

```diff
@@ -70,9 +70,9 @@ If you are using Maven, add the following dependency.
 
 ```xml
 <dependency>
-    <groupId>com.microsoft.azure</groupId>
+    <groupId>com.microsoft.spring.data.gremlin</groupId>
```

Comment:
Do we need the `.gremlin` for the groupId? As the artifactId already includes the gremlin.

Reply:
Sorry this is Readme, so we already have the groupId named with `com.microsoft.spring.data.gremlin`.

Reply:
Yes, as this repo has no reference to azure sdk, remove `azure` for misleading. We can figure out a better name in next Release.